torch/_refs/__init__.py

```diff
@@ -4049,6 +4164,7 @@ def equal(a: TensorLikeType, b: TensorLikeType) -> bool:
     return item(all(eq(a, b)))  # type: ignore[return-value]
 
 
+@register_decomposition(torch.ops.aten.norm)
```

Comment:
I don't think this is quite right. `aten.norm` does not take a `p: str`. In particular, it always computes a `vector_norm`. I think that `aten.norm` should be implemented in terms of `vector_norm`.

Reply:
went with a separate decomp/ref for now to avoid getting sidetracked